### PR TITLE
Custom download base url for spin downloads

### PR DESCRIPTION
**Pull request by rajatjindal**

### CODE REVIEW

**Review by vdice — APPROVED** [review]
Just reviewed https://github.com/fermyon/actions/pull/58/commits/27855fe3af22e761b55b2bcd6ceccac424d95c9b; the other commits are part of https://github.com/fermyon/actions/pull/57 and I assume this PR will be rebased after that merges.

LGTM -- though wondering if we want to doc the ability to use a custom download base url?  (Or is it more of a 'hidden' feature for now?)

**Comment by rajatjindal** [comment]
it is the former (hidden feature for now) :)

**Pull request merged**
